formats/protobuf/src/main/java/io/cloudevents/protobuf/ProtoSerializer.java

```diff
@@ -244,16 +246,14 @@ public CloudEvent end(CloudEventData data) throws CloudEventRWException {
 
                 // If it's a proto message we can handle that directly.
                 if (data instanceof ProtoCloudEventData) {
+
                     final ProtoCloudEventData protoData = (ProtoCloudEventData) data;
-                    final Message m = protoData.getMessage();
-                    if (m != null) {
-                        // If it's already an 'Any' don't re-pack it.
-                        if (m instanceof Any) {
-                            protoBuilder.setProtoData((Any) m);
-                        }else {
-                            protoBuilder.setProtoData(Any.pack(m));
-                        }
+                    final Any anAny = protoData.getAny();
+
+                    if (anAny != null) {
+                        protoBuilder.setProtoData(anAny);
                     }
```

Comment:
`anAny` cannot be null
```suggestion
                        protoBuilder.setProtoData(protoData.getAny());
```

Reply:
However, in this class might be acceptable to keep the check since technically the 2 are decoupled

Reply:
Modified to throw an exception if no data is present.